src/SFA.DAS.AdminService.Settings/WebConfiguration.cs

```diff
@@ -31,5 +31,7 @@ public class WebConfiguration : IWebConfiguration
         /// Gets or Sets the UseDfESignIn property.
         /// </summary>
         [JsonRequired] public bool UseDfESignIn { get; set; } = false;
```

Comment:
This comment can be removed since the property is self-explanatory. Also, the `UseDfESignIn` property in the interface has no setter, whereas this one does